src/ElmFormat.hs

```diff
@@ -222,9 +228,11 @@ main'' elmFormatVersion_ experimental_ args =
             Just config ->
                 do
                     let autoYes = Flags._yes config
-                    resolvedInputFiles <- Execute.run (Execute.forHuman autoYes) $ resolveFiles (Flags._input config)
+                    currentDirectoryElmVersion <- Execute.run (Execute.forHuman autoYes) $ FS.findElmVersion "."
```

Comment:
I'm not sure this is useful anymore versus just using 0.19 as the default version.

I kept current working directory global version detection to avoid behavior regressions, but I fail to see valid use cases for it. The only one I see is running in a 0.18 project with `--stdin` and without `--elm-version`, is it worth it?